src/electionguard-ui/ElectionGuard.UI/ViewModels/ViewKeyCeremonyViewModel.cs

```diff
@@ -49,10 +45,23 @@ public partial class ViewKeyCeremonyViewModel : BaseViewModel
     [ObservableProperty]
     private ObservableCollection<GuardianItem> _guardianList = new();
 
+    public override async Task OnAppearing()
+    {
+        await base.OnAppearing();
+
+        _timer.Tick += CeremonyPollingTimer_Tick;
```

Comment:
Question is if the OnAppearing is called more often than we think, such as leaving the app and coming back or getting a popup over it and then coming back.  Just want to make sure that we are not adding this callback too often.